requirements.txt

```
@@ -1,6 +1,6 @@
 astroid==2.0.4
 autopep8==1.3.5
-Django==2.1
+Django>=2.1.2
```

Es mejor que forcemos a esa versión en vez de cualquier versión mayor o igual:
```suggestion
Django==2.1.2
```